azure/Guidelines.md

```diff
@@ -302,8 +302,6 @@ There are 2 kinds of errors:
 
 :white_check_mark: **DO** ensure that the top-level error's `code` value is identical to the `x-ms-error-code` header's value.
 
-:white_check_mark: **DO** document the service's error code strings; they are part of the API contract.
-
```

Comment:
I think this rule belongs where it is since the six rules immediately above it all concern various aspects of the error code.

My suggestion is to move the rule at line 346 here to pull all the error code related guidelines together.

Reply:
I pushed it down because it felt weird to talk about "all the other fields" before we'd defined those fields.  Let's take a vote the next time we have a group.

Reply:
Discussed as a group and we'll keep it where it is, but move the blurb about error code grouping up near the other error code discussion.